src/main/java/uk/gov/hmcts/darts/common/config/SchedLockConfig.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.darts.common.config;
+
+import net.javacrumbs.shedlock.core.LockProvider;
+import net.javacrumbs.shedlock.provider.jdbctemplate.JdbcTemplateLockProvider;
+import net.javacrumbs.shedlock.spring.annotation.EnableSchedulerLock;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.scheduling.annotation.EnableScheduling;
+
+import javax.sql.DataSource;
+
+@Configuration
+@EnableScheduling
+@EnableSchedulerLock(defaultLockAtMostFor = "PT15M")
+@Profile("!intTest")
```

Comment:
What does this mean?

Reply:
This disables the SchedulerLock when the intTest profile runs. it was causing tests to fail